src/shared/scrapers/US/MO/index.js

```diff
@@ -198,6 +277,7 @@ const scraper = {
           }
         }
       });
+      await this._applyTestingCounts(counties);
```

Comment:
Since we have historical testing data for all three scraper time periods (0, '2020-02-22' and '2020-03-30'), the test counts get applied to each one.